charts/opensearch-operator/values.yaml

```diff
@@ -73,5 +73,5 @@ kubeRbacProxy:
     initialDelaySeconds: 10
 
   image: 
-    repository: "gcr.io/kubebuilder/kube-rbac-proxy"
+    repository: "registry.k8s.io/kubebuilder/kube-rbac-proxy"
```

Comment:
Unfortunately looks like the image is not present in the new registry (that is likely also the reason why the functional tests fail).
I did a quick check in the kubebuilder repo and it looks like they just didn't do the migration.

Suggestion: Use the original image (if I read the docs right kubebuilder just retags the image without any changes): `quay.io/brancz/kube-rbac-proxy`

Reply:
Ok, yeah saw that also.
trying that